packages/tools/src/stateManagement/segmentation/polySeg/Surface/computeAndAddSurfaceRepresentation.ts

```diff
@@ -0,0 +1,28 @@
+import { SegmentationRepresentations } from '../../../../enums';
+import { computeAndAddRepresentation } from '../computeAndAddRepresentation';
+import { computeSurfaceData } from './surfaceComputationStrategies';
+import { updateSurfaceData } from './updateSurfaceData';
+
+/**
+ * Computes and adds a surface representation for a given segmentation.
+ * @param segmentationId - The ID of the segmentation.
+ * @param options - Additional options for computing the surface representation.
+ * @param options.segmentIndices - The indices of the segments to compute the surface for.
+ * @param options.segmentationRepresentationUID - The UID of the segmentation representation to compute the surface for.
+ *
+ * @returns A promise that resolves when the surface representation is computed and added.
+ */
+export function computeAndAddSurfaceRepresentation(
+  segmentationId: string,
+  options: {
```

Comment:
These options are common enough that you should extract the type and use it consistently for all calls where it is relevant.